addons/spawning/functions/fnc_handler.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+
+[{
+    if(count GVAR(spawnQueue) < 1) exitWith {};
+
+    // Select oldest Spawn Request
+    private _entry = GVAR(spawnQueue) deleteAt 0;
+
+    // And create group or spawn unit
+    if (_entry isEqualType []) then {
+        _entry params ["_type", "_side", "_position"];
+        GVAR(currentGroup) = createGroup _side;
+            
+        {
+            _x params ["_key","_value"];
+            GVAR(currentGroup) setVariable [_key,_value];
+        } forEach [
+            ["position",_position]
```

Comment:
Should use `QGVAR(position)`, `"position"` is too generic. Maybe also name it something like `originalPosition` or `spawnPosition`?